coil-base/src/main/java/coil/util/Extensions.kt

```diff
@@ -192,3 +196,31 @@ internal inline fun ComponentRegistry.mapData(data: Any, lazySize: () -> Size):
     }
     return mappedData
 }
+
+@OptIn(ExperimentalCoilApi::class)
+internal inline operator fun MemoryCache.Key.Companion.invoke(
```

Comment:
Making these "constructors" extension functions to keep the `MemoryCache` interface as simple as possible.